modules/400-descheduler/hooks/internal/api/v1alpha1/doc.go

```diff
@@ -0,0 +1,22 @@
+//go:generate controller-gen object:headerFile=boilerplate.go.txt crd:allowDangerousTypes=true output:artifacts:config=../../../../crds
+
+/*
+Copyright 2021 Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+// +kubebuilder:object:generate=true
```

Comment:
I think this is an excellent way to generate the code, but for now, we try to avoid this pattern where it is possible. In the descheduler module, it seems that we do not need to modify CR with the Go code (the most problematic case).

Reply:
And, commands for code generation have to be written somewhere in the doc/makefile.

Reply:
Removed code generation.